conf/apache-vhost.conf

```diff
@@ -0,0 +1,17 @@
+<VirtualHost *:80>
+    # Uncomment the following line if you use a virtualenv
+    # SetEnv IHATEMONEY_VENV_PATH /path/to/your/venv
+    ServerAdmin admin@example.com
+    ServerName ihatemoney.example.com
+    WSGIDaemonProcess ihatemoney user=www-data group=www-data threads=5
```

Comment:
*disclaimer: that may be a silly suggestion, as I'm not an apache user ;-)*

 Why not just using [`python-home=/path/to/venv`](http://modwsgi.readthedocs.io/en/develop/configuration-directives/WSGIDaemonProcess.html?highlight=python-home) instead of an environment variable ? This would simplify *ihatemoney.wsgi*. Calling *activate* in *ihatemoney.wsgi* seems a bit weird to me.